Laravel.gitignore

```diff
@@ -8,9 +8,15 @@ app/storage/
 # Laravel 5 & Lumen specific
 public/storage
 storage/*.key
+/.idea
```

Comment:
I _think_ this is related to PHPStorm, which we're tracking in `Global/JetBrains.gitignore` and has more specific rules than just this. Can we drop it?